pyproject.toml

```diff
@@ -1,24 +1,105 @@
-# Example configuration for Black.
-
-# see https://raw.githubusercontent.com/ambv/black/master/pyproject.toml
-
-[tool.black]
-line-length = 88
-include = '\.pyi?$'
-exclude = '''
-/(
-    \.git
-  | \.hg
-  | \.mypy_cache
-  | \.tox
-  | \.venv
-  | \.ve
-  | _build
-  | buck-out
-  | build
-  | dist
-  | migrations
-  | env
-  | ve
-)/
-'''
+[tool.poetry]
+name = "rp-sidekick"
+version = "1.11.7"
+description = ""
+authors = ["Praekelt Foundation <dev@praekeltfoundation.org>"]
+repository = "http://github.com/praekeltfoundation/rp-sidekick"
+license = "BSD"
+readme = "README.rst"
+
+packages = [
+  { include = "config" },
+  { include = "msisdn_utils" },
+  { include = "randomisation" },
+  { include = "rp_dtone"},
+  { include = "rp_interceptors"},
+  { include = "rp_recruit"},
```

Comment:
I think we would have removed this before you merge this PR

Reply:
Sorry, not sure what you mean with "we would have".  Should I remove them?